.changeset/eleven-mice-repair.md

```diff
@@ -0,0 +1,9 @@
+---
+'@twilio-paste/alert': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/popover': patch
+'@twilio-paste/toast': patch
+'@twilio-paste/core': patch
+---
+
+Adjust close icon's hover state color, which was previously being overwritten.
```

Comment:
I'm not sure if this is the best way to format the changeset for this type of change; let me know if there's a more appropriate way!

Reply:
suggestion: It's helpful to prefix the message here with the package names in brackets so it's easier to parse the affected packages when reviewing the core changelog digest here https://paste.twilio.design/core/changelog/

Reply:
suggestion: I also think we might want to pull these into separate changesets (one per package, core in each)

Reply:
Since it is the same change message, it can be combined into this one changeset file.